docs/onboarding/Onboarding.md

```diff
@@ -0,0 +1,125 @@
+## Onboarding guide
```

Comment:
this should be a h1 heading and the other subsections should be updated accordingly